File: CC/Codigos_Fonte_Relatorio_1/micro02/micro02.pas
program micro02;

var 
    num1, num2 : integer;

begin
    writeln('Digite o primeiro número: ');
    read(num1);
    write('Digite o segundo número: ');
    read(num2); 

    if(num1 > num2) then write('O primeiro número ', num1, ' é maior que o segundo número ', num2)
    else write('O segundo número ', num2, ' é maior que o primeiro número ', num1);
     
end.

File: CC/java/micro02.java
public class Micro02{

    public static void main(String[] args){
        double num1, num2;
        System.out.println("Digite o primeiro número: ");
        num1 = Integer.parseInt(System.console().readLine());
        System.out.println("Digite o segundo número: ");     
        num2 = Integer.parseInt(System.console().readLine()); 
        
        if(num1 > num2) System.out.print("O primeiro número "+num1+" é maior que o segundo"+num2); 
        else System.out.print("O segundo número "+num2+" é maior que o primeiro"+num1);

                
    }
}
